File: fabric/src/main/java/juuxel/adorn/client/resources/ColorManagerFabric.java
package juuxel.adorn.client.resources;

import net.fabricmc.fabric.api.resource.IdentifiableResourceReloadListener;
import net.minecraft.util.Identifier;

public final class ColorManagerFabric extends ColorManager implements IdentifiableResourceReloadListener {
    public static final ColorManagerFabric INSTANCE = new ColorManagerFabric();

    @Override
    public Identifier getFabricId() {
        return ID;
    }
}

File: fabric/src/main/kotlin/juuxel/adorn/client/resources/ColorManagerFabric.kt
package juuxel.adorn.client.resources

import juuxel.adorn.AdornCommon
import net.fabricmc.fabric.api.resource.IdentifiableResourceReloadListener

object ColorManagerFabric : ColorManager(), IdentifiableResourceReloadListener {
    private val ID = AdornCommon.id("color_manager")

    override fun getFabricId() = ID
}
